NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -30,4 +44,60 @@ std::vector<GlobalIndexType> getIndices(
     return indices;
 }
 
+double norm(GlobalVector const& x, unsigned const global_component,
```

Comment:
In the TES case the global vector has components p, T and x. I want to be able to set relative and absolute error tolerances per component.
Another implementation of this functionality would be to copy the wanted components into a temporary vector and compute the norm for that. Then functionality would be more separated but it would require additional storage.